### ISSUE TITLE
Hitting the Juggle Button multiple times

### ISSUE THREAD

**Issue by geniusupgrader** [yDgunz/gunswap#34]
Hitting the Juggle Button multiple times (hit it 10-20 times for a noticable effect)
slows down the animation.
It looks like it jumps over some render times. maybe timeelapsed or something must be reset, when hitting juggle button...

**Comment by yDgunz** [comment]
Yeah there's definitely some memory mismanagement going on somewhere... I often just refresh the page, which is obviously a hack.
